test/registry/withdraw.js

```diff
@@ -42,23 +42,77 @@ contract('Registry', (accounts) => {
       const registry = await Registry.deployed();
       const listing = utils.getListingHash('shouldntwithdraw.net');
 
+      const deposit = minDeposit.plus(bigTen(1));
+
       // Whitelist, then challenge
-      await utils.addToWhitelist(listing, minDeposit, applicant);
+      await utils.addToWhitelist(listing, deposit, applicant);
       await utils.as(challenger, registry.challenge, listing, '');
 
       try {
         // Attempt to withdraw; should fail
-        await utils.as(applicant, registry.withdraw, listing, withdrawAmount);
-        assert.strictEqual(false, 'Applicant should not have been able to withdraw from a challenged, locked listing');
+        await utils.as(applicant, registry.withdraw, listing, '1');
```

Comment:
Reword it statement => 'should not withdraw tokens where the amount is less than twice the minDeposit and the listing is locked in a challenge'.